x-pack/plugin/autoscaling/src/test/java/org/elasticsearch/xpack/autoscaling/storage/ReactiveStorageDeciderDecisionTests.java

```diff
@@ -682,6 +686,7 @@ static ClusterState addDataNodes(DiscoveryNodeRole role, String prefix, ClusterS
 
     static DiscoveryNode newDataNode(DiscoveryNodeRole role, String nodeName) {
         return new DiscoveryNode(
+            nodeName,
```

Comment:
Uses node name as nodeId. This makes it easier to distinguish hot and warm nodes in routing table output.